docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,377 @@
+  toggle is enabled.
+</Admonition>
+
+## Step 5/7. Start the Database Service
+Start the Database Service:
+
+```code
+$ teleport start --config=/etc/teleport.yaml
+```
+
+## Step 6/7. Create SQL Server AD users
+
+<Admonition type="note">
+  You can skip this step if you already have Active Directory logins in your
+  SQL Server.
+</Admonition>
+
+Connect to your SQL Server as an administrative account (e.g. `sa`) and create
+logins that will use Active Directory authentication:
+
+```sql
+master> CREATE LOGIN [EXAMPLE\alice] FROM WINDOWS WITH DEFAULT_DATABASE = [master], DEFAULT_LANGUAGE = [us_english];
+```
+
+## Step 7/7. Connect
+
+Log in to your Teleport cluster. Your SQL Server database should appear in the
+list of available databases:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name      Description         Labels
+# --------- ------------------- -------
+# sqlserver                     env=dev
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+$ tsh db ls
+# Name      Description         Labels
+# --------- ------------------- -------
+# sqlserver                     env=dev
+```
+
+</ScopedBlock>
+
+To retrieve credentials for a database and connect to it:
+
+```code
+$ tsh db connect --db-user=teleport sqlserver
+```
+
+<Admonition type="note">
+  The `mssql-cli` command-line client should be available in `PATH` of the machine
+  you're running `tsh db connect` from.  
+  
+  `mssql-cli` is not required for SQL Server connections.  Use `tsh proxy db --db-user=teleport --tunnel sqlserver`
+   to connect from other DB Clients such as Microsoft SQL Server Management Studio.
+</Admonition>
+
+To log out of the database and remove credentials:
+
+```code
+$ tsh db logout sqlserver
+```
+
+## Troubleshooting
+
+### Teleport CA and CRL not imported correctly.
+
+When connecting to your database, you get an error `Error message: authentication
+failed` and Teleport database service logs have the error message `Failed to
+authenticate with KDC: kinit: Client not trusted while getting initial
+credentials`. This happens when the Teleport Database CA is not imported
+correctly or propagated yet. You can force the propagation by running
+`certutil -pulse` and trying to connect to your database.
+
+### Invalid KDC hostname.
+
+If you’re connecting to your database and receive the error `Error message:
+authentication failed` and on Teleport database service logs, there is the error
+entry `Failed to authenticate with KDC: Password for user@AD.TELEPORT.DEV:
+\nkinit: Cannot read password while getting initial credentials`, which means
+that the KDC hostname is wrong. You can verify your domain controller’s SPN to
+see if they’re set correctly and update the value on the field `kdc_hostname` on
+your database's configuration.
+
+### Teleport cannot verify database CA.
```

Comment:
This applies to also to  SQLServer integration with keytab files 
Could we updated the docs and extend the details by SQLServer TLS setup configuration ?